packages/block/CHANGELOG.md

```diff
@@ -24,21 +134,21 @@ PR [#64](https://github.com/ethereumjs/ethereumjs-block/pull/64)
 - Remove `ethereumjs-testing` dependency (much smaller dev dependencies),
   PR [#61](https://github.com/ethereumjs/ethereumjs-block/pull/61)
 
-[2.2.0]: https://github.com/ethereumjs/ethereumjs-vm/compare/%40ethereumjs%2Fvm%402.1.0...%40ethereumjs%2Fvm%402.2.0
+[2.2.0]: https://github.com/ethereumjs/ethereumjs-vm/compare/%40ethereumjs%2Fblock%402.1.0...%40ethereumjs%2Fblock%402.2.0
```

Comment:
fixed comparison link, somehow the `sed` command used during the migration substituted it erroneously.